firestore/api/SolutionCounter/SolutionCounter/Classes/Startup.cs

```diff
@@ -0,0 +1,67 @@
+﻿/*
+ * Copyright (c) 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
```

Comment:
I think, we should make tests startup in fashion of [this file](https://github.com/q-logic/dotnet-docs-samples/blob/master/firestore/api/PaginateData/Program.cs). They are calling tests from cmd (as we figured out in Backup project):
https://github.com/q-logic/dotnet-docs-samples/blob/df041dce80b8a34247c1428bbbd69cd4399d26df/firestore/api/PaginateData/Program.cs#L26-L35
And let's union `Counter.cs` and `ApiFacade.cs` into one file.

Reply:
Solved